dependencies/pom.xml

```diff
@@ -124,6 +124,7 @@
         <version.lib.mongodb.reactivestreams>1.11.0</version.lib.mongodb.reactivestreams>
         <version.lib.mssql-jdbc>8.4.1.jre8</version.lib.mssql-jdbc>
         <version.lib.neo4j>4.4.3</version.lib.neo4j>
+        <version.lib.hibernate-validator>7.0.2.Final</version.lib.hibernate-validator>
```

Comment:
The dependency versions should be ordered in alphabetical order. There seem to be a few  issues (neo4j), but in general that is the rule.

Reply:
Fixed